kbkdf/README.md

```diff
@@ -0,0 +1,72 @@
+# RustCrypto: KBKDF
+
+[![crate][crate-image]][crate-link] [![Docs][docs-image]][docs-link] ![Apache2/MIT licensed][license-image] ![Rust Version][rustc-image] [![Project Chat][chat-image]][chat-link] [![Build Status][build-image]][build-link]
```

Comment:
For better readability it's better to split the badges into multiple lines, e.g. see [here](https://raw.githubusercontent.com/RustCrypto/KDFs/refs/heads/master/hkdf/README.md).

Reply:
https://github.com/TheBestTvarynka/KDFs/pull/5

Reply:
I was planning to address comments tomorrow morning 😅 anyway, thank you! 🤗 
done